1.8.7 and 1.8.8 is broken

When I install the latest version into a project, I cannot call the styleguidist, because Node does not find the executable.

```bash
> vue-styleguidist server

env: node\r: No such file or directory
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! testme@1.0.0 styleguide: `vue-styleguidist server`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the testme@1.0.0 styleguide script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

When I downgrade to 1.8.6 I can start Styleguidist again. Is there a new command or are the new versions just broken?

--- a/.gitattributes
+++ b/.gitattributes
@@ -1,0 +1,2 @@
+*.js text eol=lf
+*.sh text eol=lf

